canonical/juju-dashboard
# WD-7715 - feat: Remove Settings page; WD-7716 - feat: Add analyticsEnabled config option

## Done

- Removed Settings page.
- Added `analyticsEnabled` config option.

## QA

- Check that the Settings page can no longer be accessed from the UI and through the `/settings` route.
- Check that `analyticsEnabled` config option is present and does what it should.

## Details

- https://warthogs.atlassian.net/browse/WD-7715
- https://warthogs.atlassian.net/browse/WD-7716
